test/functional/p2p_ibd_stalling.py

```diff
@@ -76,6 +76,7 @@ def run_test(self):
         peers = []
 
         self.log.info("Check that a staller does not get disconnected if the 1024 block lookahead buffer is filled")
+        self.mocktime = int(time.time()) + 1
```

Comment:
30705: should be use `bump_mocktime` here instead?

Reply:
this test uses `self.disable_mocktime = True`